vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -383,16 +405,20 @@ def acquire_read(self):
                     # we need to wait until it is written
 
                     # Release the processor to other threads
-                    os.sched_yield()
+                    sched_yield()
 
-                    # if we wait for a long time, we should warn the user
+                    # if we wait for a long time, log a message
                     if (time.monotonic() - start_time >
                             VLLM_RINGBUFFER_WARNING_INTERVAL * n_warning):
-                        logger.warning(
-                            "No available block found in %s second. ",
-                            VLLM_RINGBUFFER_WARNING_INTERVAL)
+                        logger.debug("No available block found in %s second. ",
```

Comment:
it seems we don't log this message at all by default now?

Reply:
@youkaichao any idea how to suppress this message when vLLM is idle? Right now on current main if we're not serving any requests, we will tell users that vLLM is probably hanging every 60 seconds 😆